feature_engine/docstrings.py

```diff
@@ -42,6 +42,14 @@ def __call__(self, obj):
         parameters or performing the transformation.
         """
 
+_input_features_docstring = """input_features: str, list, default=None
+        Input features. If `None`, then the names of all the variables in the
+        transformed dataset (original + new variables) is returned. If 'index',
+        then the name of the features created from the index is returned. If list
+        with feature names, the names for the datetime features derived from
+        the variables in the list will be returned."
+    """.rstrip()
```

Comment:
This doscstring is correct for the class datetimeFeatures that can obtain features from the index as well. All other transformers cannot.

I would make a generic docstring that applies to all classes that do not create features:

```
_input_features_docstring = """input_features: str, list, default=None
        Input features. If `None`, then the names of all the variables in the
        transformed dataset (original + new variables) is returned. If list
        with feature names, the features in the list will be returned."
    """.rstrip()
```

Also note that lines 46 to 50 need a "tab" indentation respect to current state, otherwise they appear bold in the documentation.